client/my-sites/plans-v2/plans-filter-bar/style.scss

```diff
@@ -80,3 +80,34 @@
 		}
 	}
 }
+
+.plans-filter-bar.sticky {
+	position: fixed;
+
+	// This is the height of the masterbar
+	top: 47px;
+
+	width: 100%;
+	max-width: 1040px;
+
+	z-index: 2;
+
+	@include breakpoint-deprecated( '>660px' ) {
+		width: calc( 100% - 277px );
```

Comment:
It'd be useful to mention where these numbers come from or how they've been computed.

Reply:
Excellent idea!

Reply:
We should follow the coding guidelines here, no? https://wpcalypso.wordpress.com/devdocs/docs/coding-guidelines/css.md#media-queries

Reply:
Good call. We can only replace 960px with one of Gutenberg's breakpoints, the other is not supported but we still need it.